connectors/github/element-templates/github-connector.json

```diff
@@ -245,6 +245,10 @@
         {
           "name": "Create an organization invitation",
           "value": "orgInvitation"
+        },
+        {
+          "name": "Get repository content",
+          "value": "getContent"
```

Comment:
One last nitpick:

I think "getRepoContent" would be better here now that this action lives inside the Repositories group.

My intention here is to prevent issues later if another GitHub entity (like pulls) introduces a "get content" endpoint in future.

Let me know if you have any objections to this. Otherwise, the changes look good!